react/CompositeRow/Readme.md

```diff
@@ -0,0 +1,85 @@
+```js
+const { Media, Bd, Img } = require('../Media')
+const { default: UIChip } = require('../Chip')
+const { Bold } = require('../Text')
+const { default: Icon } = require('../Icon')
+const { default: VariantInfo } = require('docs/components/VariantInfo')
+const { default: CompositeRow } = require('./index')
+
+const fnacLogoURL = 'https://upload.wikimedia.org/wikipedia/commons/2/2e/Fnac_Logo.svg';
+
+const Chip = React.memo(({ children, ...props }) => (
+  <UIChip
+    variant="outlined"
+    className='u-mr-0 u-mb-0'
+    size='small'
+    children={children} {...props}
+  />
+))
+
+const ChipImage = React.memo(({ src }) => (
+  <img className='u-mr-half' src={src} height='50%' />
+))
+
+const compositeStyle = {
+  boxShadow: '0 0 10px rgba(0, 0, 0, 0.15)',
+  boxSizing: 'border-box', 
+}
+
+const CompositeRight = () => (
+  <Img>
+    <Bold>12,15€</Bold>
+  </Img>
+)
+
+const CompositeImage = () => (
+  <Circle backgroundColor='var(--melon)'>
+    <Icon icon='wallet' />
+  </Circle>
+)
+
+const CompositeChips = () => (
+  <>
+    <Chip>
+      <ChipImage src={fnacLogoURL} /> Fnac
+    </Chip>
+    <Chip>
+      <ChipImage src={fnacLogoURL} /> Fnac
+    </Chip>
+  </>
+)
+
+initialState.variants = [
+  { chips: true, secondaryText: true, dots: true, icon: true },
+  { chips: false, secondaryText: true, icon: true, dots: true },
+  { shrinked: true, chips: false, secondaryText: false, dots: false, icon: true },
+  { chips: false, secondaryText: false, dots: false, icon: false },
+  { shrinked: true, chips: false, secondaryText: false, dots: false, icon: false }
+];
+
+const onChangeVariant = (updatedVariant, i) => {
+  const newVariants = [
+    ...state.variants.slice(0, i),
+    updatedVariant,
+    ...state.variants.slice(i + 1)
+  ]
+  setState({ variants: newVariants })
+};
```

Comment:
Not blocking, but I wonder if having only one interactive example wouldn't be better then a series of variants that are all just different default options. As a dev I think I'd rather have ust one example to play with, but curious to see what others think.

Reply:
OK for one interactive example to play with, I was mainly worried about getting various examples into screenshots, and it was also easier when  developing to see all possible variants in one shot.

- For testing, it is possible to check `isTesting()` so we can show all variants in one go
- For developing, I think we should have a way to quickly toggle all variants (by modifyying the source code)

What I suggest is another `<Variants />` helper that can show either the first variant or either all variants depending on a prop.

Reply:
For information, I just saw that https://material-ui.com/components/lists/#interactive is showing 4 examples at once. Do you have any preference @drazik, @Crash-- ?

Reply:
Personally, the more exemples I can have, the more exemples I take. It helps me to really think about what variante I should use instead of having to test them before

Reply:
I also think we should have more than one example like MUI does.

Reply:
Ok let's leave it with multiple examples, looks like I'm in the minority here and it's mostly a matter of taste I guess :)

Reply:
Done